### PR TITLE
Add Codes and files for the project

### PR DESCRIPTION
**Pull request by yash244466666**
Add code to the following files for completing the project :
[index.html](https://github.com/yash244466666/Awesome-books-ES6-M2W2/blob/AddProjectCodesAndFiles/index.html)
[index.js](https://github.com/yash244466666/Awesome-books-ES6-M2W2/blob/AddProjectCodesAndFiles/index.js)
[index.css](https://github.com/yash244466666/Awesome-books-ES6-M2W2/blob/AddProjectCodesAndFiles/index.css)
[store.js](https://github.com/yash244466666/Awesome-books-ES6-M2W2/blob/AddProjectCodesAndFiles/modules/store.js)
[nav-menu.js](https://github.com/yash244466666/Awesome-books-ES6-M2W2/blob/AddProjectCodesAndFiles/modules/nav-menu.js)
[luxon.js](https://github.com/yash244466666/Awesome-books-ES6-M2W2/blob/AddProjectCodesAndFiles/modules/luxon.js)
[Book-storage.js](https://github.com/yash244466666/Awesome-books-ES6-M2W2/blob/AddProjectCodesAndFiles/modules/Book-storage.js)

Add the following files as recourses for the project : 
[books-bg.png](https://github.com/yash244466666/Awesome-books-ES6-M2W2/blob/AddProjectCodesAndFiles/res/books-bg.png)
[bg.jpg](https://github.com/yash244466666/Awesome-books-ES6-M2W2/blob/AddProjectCodesAndFiles/res/bg.jpg)

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  color: #0e3f6a;
+  margin-top: 166px;
+}
+
+#top-heading {
+  color: #000;
+  font-family: 'Courier New', Courier, monospace;
+  margin-left: 50vw;
+
+  /* width: 25vw; */
```
Comment (christianbiring1) [inline index.css]: - [ ] Kindly make sure you dod not have some piece of code in your code base for it's not a best practice for clean code

File: index.html [hunk 2]
```diff
@@ -1,13 +1,64 @@
...
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
```
Comment (christianbiring1) [inline index.html]: - [ ] Kindly consider using the correct path to link your `style.css` to your HTML file.
- [ ] Also you may consider fixing your style issues as you can notice how your page is not presentable, and also the error in the console in the `bubble.js`
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @yash244466666 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] The ES6 modules are used properly :+1: 
- [ ] Arrow functions were used

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Give your PR a descriptive `title` and `summary` telling what changes were introduced in the project.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (yash244466666) [reply index.html]: @christianbiring1 Sir, I've found a solution for this error at https://stackoverflow.com/questions/74281137/companion-bubble-js-in-the-console this is a bug from the loom chrome extension .

**Comment by yash244466666** [comment]
Add changes to the following files as instructed. 
[index.html](https://github.com/yash244466666/Awesome-books-ES6-M2W2/blob/AddProjectCodesAndFiles/index.html)
[index.css](https://github.com/yash244466666/Awesome-books-ES6-M2W2/blob/AddProjectCodesAndFiles/index.css)

And the error on the console was a bug from "**Loom chrome Extension**" https://stackoverflow.com/questions/74281137/companion-bubble-js-in-the-console

File: index.css (hunk 1 above)
Comment (peterrobert) [reply index.css]: This change has been done well

File: index.html (hunk 2 above)
Comment (peterrobert) [reply index.html]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @yash244466666 , 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**